controllers/membersController.js

```diff
@@ -11,9 +11,15 @@ const getMembers = async (req, res) => {
   try {
     const allMembers = await memberQuery.fetchMembers()
 
+    if (allMembers.length) {
```

Comment:
@swarajpure , any reason we want these separate messages?
We can rely on the array length at the UI, right?

Reply:
I guess it would be better to have a different message in case of no member found. Personally, it doesn't make sense to me where the `members` array is empty but still the message says `Members returned successfully!`, since there are no members returned. Again, my opinion. Let me know what you think about it 🙂